### ISSUE TITLE
Exclude entities in commented out parts of files

### ISSUE THREAD

**Issue by renarena** [dummylabs/thewatchman#85]
I get warnings about entity ids only used in commented out yaml parts.
F.e. in customize.yaml i have commented out a broken device 
```
  #~ - entity_id: "sensor.device_energy_total"
    #~ unit_of_measurement: 'kWh'
    #~ device_class: energy
    #~ state_class: total_increasing
```

**Comment by kepath** [comment]
I have created a pull request to fix this
https://github.com/dummylabs/thewatchman/pull/124

**Comment by dummylabs** [comment]
Will be shipped with 0.6.4, thanks.
